test/api/v3/integration/groups/PUT-groups.test.js

```diff
@@ -0,0 +1,50 @@
+import {
+  generateUser,
+  translate as t,
+} from '../../../../helpers/api-integration.helper';
+
+describe('Put /group', () => {
+  let groupLeader;
+  let groupName = 'Test Public Guild';
+  let groupType = 'guild';
+
+  beforeEach(async () => {
+    groupLeader = await generateUser({balance: 1});
+  });
+
+  xit('returns an error when a non group leader tries to update', async () => {
```

Comment:
Sorry, missed the note and dove right into the code. 

Why is this dependent on the invite route? Can't we just update a user to be part of the group?